internal/controller/kuadrant_controller_test.go

```
@@ -0,0 +1,164 @@
+
+	// Create events slice
+	events := []controller.ResourceEvent{
+		{EventType: controller.UpdateEvent},
+	}
+
+	// Schedule retry
+	kuadrantController.ScheduleRetry(100*time.Millisecond, events)
+
+	// Modify the original events slice (simulate backing array reuse)
+	events[0].EventType = controller.DeleteEvent
+	events = append(events, controller.ResourceEvent{EventType: controller.CreateEvent})
```

golangci-lint (2.12.2)

[error] ineffectual assignment to events

(ineffassign)